### PR TITLE
Corrected script for some item

### PR DESCRIPTION
**Pull request by idk-whoami**
* **Addressed Issue(s)**: 

* **Server Mode**: re

* **Description of Pull Request**:

### CODE REVIEW

File: db/re/item_db_equip.yml [hunk 1]
```diff
@@ -131572,6 +132286,41 @@ Body:
...
+         bonus bDelayRate,-((readparam(bLuk)+readparam(bStr))/18);
+         bonus2 bSubEle,Ele_Holy,(.@r>=12?5:3)*((readparam(bLuk)+readparam(bStr))/18);
+         bonus2 bSubEle,Ele_Dark,(.@r>=12?5:3)*((readparam(bLuk)+readparam(bStr))/18);
+         bonus bAspdRate,((readparam(bAgi)+readparam(bVit))/18);
+         bonus bMaxHPrate,(.@r>=12?.@r:0)+((readparam(bAgi)+readparam(bVit))/18);
+         bonus2 bSubEle,Ele_Fire,(.@r>=12?5:3)*((readparam(bAgi)+readparam(bVit))/18);
+         bonus2 bSubEle,Ele_Water,(.@r>=12?5:3)*((readparam(bAgi)+readparam(bVit))/18);
+         bonus bVariableCastrate,-((readparam(bInt)+readparam(bDex))/18);
+         bonus2 bSubEle,Ele_Wind,(.@r>=12?5:3)*((readparam(bInt)+readparam(bDex))/18);
+         bonus2 bSubEle,Ele_Earth,(.@r>=12?5:3)*((readparam(bInt)+readparam(bDex))/18);
```
Comment (Atemo) [inline db/re/item_db_equip.yml:132322]:
```suggestion
        .@a = (readparam(bLuk)+readparam(bStr))/18;
        .@b = (readparam(bAgi)+readparam(bVit))/18;
        .@c = (readparam(bInt)+readparam(bDex))/18;
         .@bonus = (.@r>=12 ? 5 : 3);
         bonus bDelayRate,-.@a;
         bonus2 bSubEle,Ele_Holy,.@bonus*.@a;
         bonus2 bSubEle,Ele_Dark,.@bonus*.@a;
         bonus bAspdRate,.@b;
         bonus bMaxHPrate,(.@r>=12?2:1)*.@b;
         bonus2 bSubEle,Ele_Fire,.@bonus*.@b;
         bonus2 bSubEle,Ele_Water,.@bonus*.@b;
         bonus bVariableCastrate,-.@c;
         bonus2 bSubEle,Ele_Wind,.@bonus*.@c;
         bonus2 bSubEle,Ele_Earth,.@bonus*.@c;
```

File: db/re/item_db_usable.yml [hunk 2]
```diff
@@ -2272,6 +2272,93 @@ Body:
...
+      NoCart: true
+      NoGuildStorage: true
+      NoMail: true
+      NoAuction: true
+    Script: |
+      .@type = getiteminfo(getequipid(EQI_HAND_R), ITEMINFO_VIEW);
+      if (.@type == W_1HAXE || .@type == W_2HAXE || .@type == W_MACE) {
+         skilleffect "BS_ADRENALINE",0;
+         sc_start SC_ADRENALINE,150000,5;
+      }
```
Comment (Atemo) [inline db/re/item_db_usable.yml:2298]: I don't know what the correct implementation of this item should be, but I believe it should look like this one instead
```
  - Id: 12471
    AegisName: F_Adrenaline_Scroll
    Name: F Adrenaline Scroll
    Type: Delayconsume
    Buy: 2
    Weight: 10
    Script: |
      itemskill "BS_ADRENALINE",5;
```

File: db/re/item_db_usable.yml [hunk 3]
```diff
@@ -35035,6 +35122,351 @@ Body:
...
+      NoDrop: true
+      NoTrade: true
+      NoSell: true
+      NoCart: true
+      NoGuildStorage: true
+      NoMail: true
+      NoAuction: true
+    Script: |
+      skilleffect "AL_BLESSING",0;
+      sc_start SC_BLESSING,240000,10;
```
Comment (Atemo) [inline db/re/item_db_usable.yml:35205]: Same, for me the item should look like this one
```
  - Id: 12522
    AegisName: E_Blessing_10_Scroll
    Name: Blessing Scroll Lv 10
    Type: Delayconsume
    Buy: 2
    Weight: 10
    Trade:
      Override: 100
      NoDrop: true
      NoTrade: true
      NoSell: true
      NoCart: true
      NoGuildStorage: true
      NoMail: true
      NoAuction: true
    Script: |
      itemskill "AL_BLESSING",10;
```

File: db/re/item_db_usable.yml [hunk 4]
```diff
@@ -35035,6 +35122,351 @@ Body:
...
+      NoCart: true
+      NoGuildStorage: true
+      NoMail: true
+      NoAuction: true
+    Script: |
+      if (Hp>15) {
+         skilleffect "AL_INCAGI",0;
+         sc_start SC_INCREASEAGI,240000,10;
+         heal -15,0;
+      }
```
Comment (Atemo) [inline db/re/item_db_usable.yml:35229]: Same idea as above

**Review by Atemo — CHANGES_REQUESTED** [review]

File: db/re/item_combo_db.txt [hunk 5]
```diff
@@ -13,7 +13,7 @@
 1269:5467,{ bonus bAspd,1; bonus bFlee2,3; }
 1284:2892,{ bonus2 bSkillAtk,"AS_SONICBLOW",50; bonus bBaseAtk,50; bonus bFlee,-30; }
 1285:2892,{ bonus bCritical,4; bonus bCritAtkRate,40; bonus bMaxHPRate,-10; }
-1287:15123,{ if (getequiprefinerycnt(EQI_HAND_R) >= 7) { autobonus "{ bonus bIgnoreDefClass,Class_Normal; }",1000,3000,BF_SHORT|BF_NORMAL; } /* Confirm: Success rate */ }
+1287:15123,{ if (getequiprefinerycnt(EQI_HAND_R) >= 7) { autobonus "{ bonus bIgnoreDefClass,Class_Normal; }",70,3000,BF_SHORT|BF_NORMAL; } }
```
[the remark this answers is not in the document]
Comment (idk-whoami) [reply db/re/item_combo_db.txt]: Sorry, I think like https://www.divine-pride.net/database/item/15248

File: db/re/item_db_etc.yml [hunk 6]
```diff
@@ -2690,7 +2690,7 @@ Body:
       DropEffect: CLIENT
     Script: |
       bonus bLuk,1;
-      bonus bCritical,1;
+      bonus bCriticalRate,1;
```
[the remark this answers is not in the document]
Comment (idk-whoami) [reply db/re/item_db_etc.yml]: Thanks!

File: db/re/item_combo_db.txt [hunk 7]
```diff
@@ -2492,3 +2492,23 @@
 32300:19451,{ bonus2 bSubRace,RC_Player_Human,3; bonus2 bAddEle,Ele_All,15; bonus2 bMagicAddEle,Ele_All,15; }
 32300:19452,{ bonus2 bSubRace,RC_Player_Human,3; bonus2 bAddEle,Ele_All,15; bonus2 bMagicAddEle,Ele_All,15; }
 32300:27301,{ bonus bAspdRate,10; bonus2 bAddRace,RC_All,10; }
+2965:18797,{ bonus bAllStats,1; bonus bMaxHPrate,5; bonus bMaxSPrate,5; bonus3 bAutoSpellWhenHit,"WZ_VERMILION",5,5; bonus2 bAddRace,RC_All,10; bonus bVariableCastrate,-10; }
+2965:18798,{ bonus bAllStats,1; bonus bMaxHPrate,5; bonus bMaxSPrate,5; bonus3 bAutoSpellWhenHit,"WZ_VERMILION",5,5; bonus2 bAddRace,RC_All,10; bonus bVariableCastrate,-10; }
```
[the remark this answers is not in the document]
Comment (idk-whoami) [reply db/re/item_combo_db.txt]: Thanks!

File: db/re/item_combo_db.txt [hunk 8]
```diff
@@ -2492,3 +2492,23 @@
...
+5891:2269,{ bonus bAllStats,1; }
+5891:2270,{ bonus bAllStats,1; }
+18906:4077,{ bonus bHealPower,2; bonus bVariableCastrate,-15; }
+18912:2254,{ bonus2 bExpAddClass,Class_All,5; }
+19221:20866,{ bonus bAllStats,1; }
+22105:2655,{ bonus bBaseAtk,50; bonus2 bAddDefMonster,1196,20; bonus2 bAddDefMonster,1197,20; }
+22105:28388,{ bonus bBaseAtk,50; bonus2 bAddDefMonster,1196,20; bonus2 bAddDefMonster,1197,20; }
+22105:15040,{ .@r = getequiprefinerycnt(EQI_SHOES); bonus bBaseAtk,5+.@r; }
+22105:2913,{ bonus bBaseAtk,getequiprefinerycnt(EQI_SHOES); }
+28388:16053,{ autobonus "{ bonus2 bIgnoreDefRaceRate,RC_All,100; }",5,(getequiprefinerycnt(EQI_SHOES)>=5)?6000:5000,BF_NORMAL,"{}"; }
```
[the remark this answers is not in the document]
Comment (idk-whoami) [reply db/re/item_combo_db.txt]: Thanks!

File: db/re/item_db_equip.yml [hunk 9]
```diff
@@ -68173,6 +68233,30 @@ Body:
...
+      Alchemist: true
+      Blacksmith: true
+      Crusader: true
+      Knight: true
+    Classes:
+      All_Third: true
+    Locations:
+      Right_Hand: true
+    WeaponLevel: 4
+    EquipLevelMin: 150
```
[the remark this answers is not in the document]
Comment (idk-whoami) [reply db/re/item_db_equip.yml:68256]: Thanks!

File: db/re/item_db_equip.yml [hunk 10]
```diff
@@ -74173,6 +74257,29 @@ Body:
...
+    Slots: 1
+    Locations:
+      Head_Top: true
+      Head_Mid: true
+      Head_Low: true
+    EquipLevelMin: 88
+    Refineable: true
+    View: 868
+    Script: |
+      .@r = getrefine();
```
[the remark this answers is not in the document]
Comment (idk-whoami) [reply db/re/item_db_equip.yml:74276]: Thanks!

File: db/re/item_db_equip.yml [hunk 11]
```diff
@@ -76861,6 +77074,46 @@ Body:
...
+    View: 1092
+    Script: |
+      bonus bInt,1;
+      bonus bDex,1;
+      if (readparam(bInt) >= 120)
+         bonus bMatk,10;
+      else if (readparam(bInt) >= 99)
+         bonus bMatk,7;
+      else
+         bonus bMatk,5;
```
[the remark this answers is not in the document]
Comment (idk-whoami) [reply db/re/item_db_equip.yml]: Thanks!

File: db/re/item_db_equip.yml [hunk 12]
```diff
@@ -77697,6 +77993,28 @@ Body:
...
+      Novice: false
+      SuperNovice: false
+    Classes:
+      All_Upper: true
+    Locations:
+      Garment: true
+    EquipLevelMin: 70
+    Refineable: true
+    Script: |
+      bonus bMaxHPrate,5;
```
[the remark this answers is not in the document]
Comment (idk-whoami) [reply db/re/item_db_equip.yml:78014]: Thanks!

File: db/re/item_db_equip.yml [hunk 13]
```diff
@@ -91505,6 +91929,36 @@ Body:
...
+    Defense: 10
+    Slots: 1
+    Locations:
+      Garment: true
+    EquipLevelMin: 70
+    Refineable: true
+    Script: |
+      bonus2 bSubEle,Ele_Fire,10;
+      bonus2 bSubEle,Ele_Water,-10;
+      bonus bShortWeaponDamageReturn,10+(getrefine()*2);
```
[the remark this answers is not in the document]
Comment (idk-whoami) [reply db/re/item_db_equip.yml:91961]: Thanks!

File: db/re/item_db_equip.yml [hunk 14]
```diff
@@ -91505,6 +91929,36 @@ Body:
...
+    Defense: 10
+    Slots: 1
+    Locations:
+      Garment: true
+    EquipLevelMin: 70
+    Refineable: true
+    Script: |
+      bonus2 bSubEle,Ele_Wind,10;
+      bonus2 bSubEle,Ele_Earth,-10;
+      bonus bLongAtkDef,5+getrefine();
```
[the remark this answers is not in the document]
Comment (idk-whoami) [reply db/re/item_db_equip.yml:91946]: Thanks!

File: db/re/item_db_equip.yml (hunk 1 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml]: Thanks!

File: db/re/item_db_usable.yml (hunk 3 above)
Comment (idk-whoami) [reply db/re/item_db_usable.yml]: Thanks!

File: db/re/item_db_usable.yml (hunk 4 above)
Comment (idk-whoami) [reply db/re/item_db_usable.yml]: Thanks!

File: db/re/item_db_usable.yml (hunk 2 above)
Comment (idk-whoami) [reply db/re/item_db_usable.yml]: Thanks!

File: db/re/item_db_equip.yml [hunk 15]
```diff
@@ -77516,6 +77785,33 @@ Body:
...
+      bonus bMaxSP,50;
+  - Id: 19011
+    AegisName: Little_Devil_S_Horn
+    Name: Little Devil's Horn
+    Type: Armor
+    Weight: 1000
+    Locations:
+      Head_Top: true
+    EquipLevelMin: 1
+    Refineable: true
```
[the remark this answers is not in the document]
Comment (idk-whoami) [reply db/re/item_db_equip.yml:77810]: Thanks!

**Comment by Atemo** [comment]
Thanks !

**Pull request merged**